fizz-buzz-tree/fizz-buzz-tree.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+let preOrderStr = '';
+const preOrderTraversal = (rootNode) => {
+  // left - root - right=
+  if (!rootNode) {
+    return undefined;
+  }
+
+  if (rootNode.value % 3 === 0 && rootNode.value % 5 === 0) {
```

Comment:
This entire module looks good. One trick with the fizzbuzz problem is that you can just multiply 3 and 5 together to hit this if condition, i.e.:

```
if (rootNode.value % 15 ==== ) {
   // logic here
}
```